### ISSUE TITLE
Take this repo down

### ISSUE THREAD

**Issue by davidmontgom** [ozgur/python-firebase#73]
Has not been updates for years and provides confusion that are new.  This DOES not work.  Else provide a warning that this repo is broken

**Comment by mikelambert** [comment]
Take this down in favor of what? Is there a better solution out there for python usage? (And arguably, the complaint should be with Google's Firebase documentation that points here, right?)

**Comment by ParitoshBh** [comment]
Pyrebase does the basic stuff pretty well! I miss monitoring the "/.info/connected" though (for desktop client).

**Comment by NullDivision** [comment]
I agree with @davidmontgom and @ParitoshBh. This package is sorely dated and should probably be abandoned at this point unless @ozgur updates the authentication module and reworks persistent connections with the newer implementations.

Was checking out Pyrebase and seems to be a more up to date route. Latest commit is also only 20 days old so...

**Comment by mikelambert** [comment]
You referring to the Pyrebase whose README says "written for python 3 and will not work correctly with python 2"? (Not an option for me, at this point in time.)

**Comment by ParitoshBh** [comment]
@NullDivision The package may not be updated but the author isn't under any obligation to do so quite frankly. And taking it down will do no good as well (especially for those who might want to carry forward from here).

And @mikelambert is right. There might be people dependent on this package and/or not willing to move to a different one. Moreover, if one package serves the purpose well over other then what's the problem? ;)

**Comment by bryanstrader** [comment]
I'm glad I finally looked here, was trying to use this repo but authentication is completely broken. Can someone point me to a better alternative for accessing Firebase via Python?

**Comment by mikelambert** [comment]
Python 2 (nothing that I know of)? Or Python 3 (check pyrebase)?

**Comment by udsharma1506** [comment]
Any better solution for python 2?
how are these packages?[ link1](https://github.com/mikexstudios/python-firebase) link2(https://github.com/shariq/firebase-python)

**Comment by mikelambert** [comment]
Did you look at these packages you mentioned? Or did you just list them and hope someone else would?
- link1 is 2013, so I assume very dead
- link2 looks like a get/put/patch API around firebase, but without any higher-level constructs

There's also https://firebase.googleblog.com/2017/04/bringing-firebase-admin-to-python.html , but at the moment it's just Custom Token Minting and ID Token Verification, though might include more going forward. But looks like they're aiming to be Admin SDKs only (which is fine for me, but perhaps not for others if you need clientside python)

**Comment by cricket007** [comment]
@mikelambert Seems the database is now available.  

https://firebase.googleblog.com/2017/07/accessing-database-from-python-admin-sdk.html

**Comment by samdobson** [comment]
Third party volunteer support for such a complex and fast-moving service as firebase is a tough ask.

The official python admin SDK now looks pretty good and may be the way to go for many...

https://github.com/firebase/firebase-admin-python
